crypto/keys/ecdsa/ecdsa_pubkey_test.go

```diff
@@ -0,0 +1,154 @@
+package ecdsa
```

Comment:
```suggestion
package ecdsa_test
```

Reply:
These are unit tests.

Reply:
Yes, I got that. In the Cosmos SDK we have the convention to build and run tests in separate `_test` packages. If you need access `ecda` package's internals, then I'd suggest to rename this file to `ecdsa_internal_test.go`

Reply:
Unresolving this comment.

Reply:
OK - will rename the file. 
PS: I don't see that this is a standard in Cosmos, nor I don't know any other bigger project which uses it.